packages/postgres-database/tests/test_clusters.py

```diff
@@ -38,21 +39,26 @@ async def user_group_id(pg_engine: Engine, user_id: int) -> int:
 
 
 @pytest.fixture
-async def create_cluster(pg_engine: Engine) -> Callable[..., Awaitable[int]]:
+async def create_cluster(
+    pg_engine: Engine, faker: Faker
+) -> AsyncIterable[Callable[..., Awaitable[int]]]:
```

Comment:
TIP: there is little value in adding annotations to the returned types in ``pytest.fixture``s because the static type-checker does not verify the arguments on the test function. It is rather more useful if

```python

@pytest.fixture
async def create_cluster( ... ):
    # annotate arguments but not return value here (except if in separate file)

async def test_can_create_cluster_with_owner(
    user_group_id: int, create_cluster: Callable[..., Awaitable[int]]
):
    # annotate arguments here 
    # static analysis will help here
```

Reply:
I agree but usually when I look for the annotations I like it when they are defined on the fixture...